src/public/cards/importFeedback.ts

```diff
@@ -25,11 +25,13 @@ export class CSVImportFeedback {
   }
 
   setFeedback(feedback: ImportItemsResponse, isEdge?: boolean) {
+    this.importStatus.classList.remove('success', 'failed', 'incomplete')
     const total = feedback.success + (feedback.failed ? feedback.failed : 0);
     const item = isEdge ? 'edges' : 'nodes';
     let errors = '';
-    Object.entries(feedback.error || {}).forEach(([key, value]) => {
-      errors += key + '\n' + `${value.length === 1 ? 'Row' : 'Rows'}: ${value.join(', ')} \n\n`;
+    Object.entries(feedback.error || {}).forEach(([key, value], index) => {
+      const jumpLine = Object.entries(feedback.error || {}).length === index + 1 ? '.' : ' \n\n';
+      errors += key + '\n' + `${value.length === 1 ? 'Row' : 'Rows'}: ${value.join(', ')}${jumpLine}`;
```

Comment:
why a const here ? and where do you use it ?

Reply:
jumpLine is used in line 34. it is used to check if it is the last error type , if it is the case we don't jump line if not we do.

check point number 10 in the document of design review.